staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/width.go

```diff
@@ -83,6 +85,11 @@ func (e WorkEstimatorFunc) EstimateWork(r *http.Request, flowSchemaName, priorit
 	return e(r, flowSchemaName, priorityLevelName)
 }
 
+type WorkEstimator interface {
```

Comment:
Don't we have a linter that demands comments on public members?

Reply:
Idk (probably not) but added comments.